scripts/install/production/roles/tgt/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
+#Install tgt and its necessary dependencies
+
+- name: Install packages for tgt CentOS installation
+  package: name={{ item }} state=latest
+  with_items:
+   - gcc
+   - cpan
+   - make
+   - firewalld
```

Comment:
No. Firewalld just manages iptables rules for you in CentOS. Why would you outright disable it? And Dan tested this setup, so it definitely works.

Reply:
it is dropping DHCP request during BMI provision

Reply:
I tested in kumo

Reply:
but that has nothing to do with firewalld itself. It only manages iptables rules.

If the firewalld service is disabled on the machine (by default, it's enabled on centos), then we have to directly make changes to iptables. But the problem with that is those changes aren't saved since there's no iptables.service on centos anymore (you have to run the iptables command everytime on boot, or add to rc.local).

See if firewalld is running, and then see if iptables has rules for dhcp port(s)?

Reply:
I suppose enabling port 67 and 68 should work even if firewalld is running. Needs to be tested tough.